exporter/graph/experiment.py

```diff
@@ -18,3 +19,10 @@ def extend(self, graph: 'ExperimentGraph'):
             self.nodes.add_node(node)
 
         return self
+
+    @staticmethod
+    def from_supplementary_files_info(supplementary_files_info, project):
+        graph = ExperimentGraph()
+        graph.nodes.add_nodes(supplementary_files_info.files + [project])
+        suppl_files_link = GraphCrawler.supplementary_file_link_for(supplementary_files_info)
+        graph.links.add_link(suppl_files_link)
```

Comment:
I think this is the change that introduced the circular dependancy causing the tests to fail.